ACE/ace/CDR_Base.inl

```diff
@@ -145,7 +145,11 @@ ACE_CDR::swap_8 (const char* orig, char* target)
     && !defined(ACE_LACKS_INLINE_ASSEMBLY)
   ACE_REGISTER unsigned long x =
     * reinterpret_cast<const unsigned long*> (orig);
+#if defined (__MINGW64__)
```

Comment:
Correct, that's what I said in the most recent comment.  Since the code is not relevant to MinGW it should not be changed in this pull request.

I also tried building the master branch using "x86_64-w64-mingw32-g++ (GCC) 4.8.2" on Ubuntu 14.04.  This didn't get very far because the special pthread.h for MinGW defines strok_r as a macro which confuses ACE's code.  The same pthread.h in *native* MinGW doesn't have this problem.